flow/e2e/bigquery/bigquery_helper.go

```diff
@@ -452,3 +452,15 @@ func (b *BigQueryTestHelper) CreateTable(tableName string, schema *model.QRecord
 
 	return nil
 }
+
+func (b *BigQueryTestHelper) RunIntQuery(query string) (int, error) {
```

Comment:
```suggestion
func (b *BigQueryTestHelper) RunInt64Query(query string) (int64, error) {
```
No need to convert int64 result to int